packages/flutter_tools/lib/src/commands/apk.dart

```diff
@@ -0,0 +1,201 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:logging/logging.dart';
+import 'package:path/path.dart' as path;
+
+import 'build.dart';
+import 'flutter_command.dart';
+
+const String _kDefaultAndroidManifestPath = 'AndroidManifest.xml';
+const String _kDefaultOutputPath = 'app.apk';
```

Comment:
I wonder if we should move all these build products into an "out" or a "build" directory by default so they don't clutter up the developer's project directory.  That will also make it easier for the end developer to git ignore them all at once.